lib/mnesia/test/ext_test.erl

```diff
@@ -248,6 +249,18 @@ select(Alias, Tab, Ms, Limit) ->
     ?DBG({Alias, ext_test_server:tab_to_list(Tab), Ms, Limit}),
     call({?FUNCTION_NAME, Alias, Tab, Ms, Limit}).
 
+select_reverse(Continuation) ->
+    ?DBG(Continuation),
+    call({?FUNCTION_NAME, Continuation}).
```

Comment:
Since you removed mnesia_lib:db_select_rev_cont this will never be called, I think you can remove it.

Reply:
I've removed it, you can check @Bentheburrito